recipe/bld.bat

```diff
@@ -0,0 +1,24 @@
+mkdir build
+cd build
+
+rem rm /mingw-w64/bin/make.exe
+bash -c "CXXDEFS=-DCOIN_USE_MUMPS_MPI_H ADD_CXXFLAGS='-I/mingw-w64/include -L/mingw-w64/lib -L/mingw-w64/bin' ../configure --build=x86_64-w64-mingw32 --disable-linear-solver-loader --with-blas-lib='-lopenblas' --with-mumps-lib='-ldmumps -lmumps_common -lpord -lgfortran' --enable-dependency-linking --prefix=/mingw-w64"
```

Comment:
disabling the linear solver loader is a bad idea - that's important functionality if you care at all about performance

Reply:
I'll take another look. I was getting unrelenting segmentation faults trying to get the linear solver loader to work, but I'm fully willing to admit the possibility of user error.

Reply:
It can be picky about finding the runtime libraries IIRC, and how the solver you're trying to load was built. I've forgotten what the status is of HSL providing Windows binaries to academic users. Dynamically loading the MKL version of Pardiso may also be flaky